ui/apps/platform/src/Containers/AccessControl/PermissionSets/PermissionSetsList.tsx

```diff
@@ -92,6 +95,11 @@ function PermissionSetsList({
                                         entityName={name}
                                     />
                                 </Td>
+                                <Td dataLabel="Origin">
+                                    <Label color={originLabelColours[getTraitsOriginLabel(traits)]}>
+                                        {getTraitsOriginLabel(traits)}
+                                    </Label>
```

Comment:
If all access control entities will render this information on form and table, what do you think about a follow up  to factor out as something like `<TraitsOriginLabel traits=(permissionSet.traits} />`

And then might it be possible to render with initially undefined value even for create case above in form?

Reply:
Factored out `TraitsOriginLabel`